src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+import java.util.ArrayList;
+
+/**
+ * Duke is a Chatbot that helps you keep track of your tasks.
+ * In this version, Duke is renamed to Nano.
+ */
+public class Duke {
+
+    /** The storage that is used to save and load the user's tasks. */
+    private static Storage storage;
+
+    /** The task list that is used to store the user's tasks. */
+    private TaskList tasks;
+
+    /** The UI to interact with the User. */
+    private Ui ui;
+
+    /** The file path to store the User's Task Data. */
+    private static final String FILEPATH = "./data/duke.txt";
+
+    /**
+     * Handles the Chatbot functionality.
+     */
+    private void handleCommand() {
+        String commandString;
+        Command command;
+
+        while (true) {
+            commandString = ui.getInput();
+            Parser parseLine = new Parser(commandString);
+            command = parseLine.getCommand();
+
+            if (command == null) {
+                ui.printInvalidCommandError();
+                continue;
+            }
+
+            try {
+                switch (command) {
+                    case BYE:
+                        ui.printBye();
+                        return;
+                    case LIST:
+                        ui.printOutput(tasks.formatList());
+                        break;
+                    case MARK:
```

Comment:
Perhaps you could include a `//fallthrough` comment, for example
```suggestion
                    case MARK: //fallthrough
```